### PR TITLE
Added backend for joinRoomButton

### PR DESCRIPTION
**Pull request by Musealo**
If the user is using a code now to join a room, they'll be pushed in an array "players" in my backend

### CODE REVIEW

File: pages/api/rooms/[room_id]/players.js [hunk 1]
```diff
@@ -61,6 +61,28 @@ export default async function handler(request, response) {
...
+            .json({ error: 'Already active in the Room' });
+        }
+        let modifyRoom = await Room.findByIdAndUpdate(
+          room._id,
+          {
+            $push: {
+              players: session.user.id,
+            },
+          },
+          { returnDocument: 'after', runValidators: true }
```
Comment (SarmadAD) [inline pages/api/rooms/[room_id]/players.js:80]: and this line also

Comment (SarmadAD) [inline pages/api/rooms/[room_id]/players.js:80]: and this line also

File: pages/api/rooms/[room_id]/players.js [hunk 2]
```diff
@@ -61,6 +61,28 @@ export default async function handler(request, response) {
...
+
+      case 'PATCH':
+        let room = await Room.findById(request.query.room_id).exec();
+
+        if (room.players.includes(session.user.id)) {
+          return response
+            .status(400)
+            .json({ error: 'Already active in the Room' });
+        }
+        let modifyRoom = await Room.findByIdAndUpdate(
```
[the remark this answers is not in the document]
Comment (Musealo) [reply pages/api/rooms/[room_id]/players.js]: Thanks

**Review by SarmadAD — APPROVED** [review]

File: pages/api/rooms/[room_id]/players.js (hunk 1 above)
Comment (Musealo) [reply pages/api/rooms/[room_id]/players.js]: Thanks

**Pull request merged**